db/compaction_picker_universal.cc

```diff
@@ -342,8 +342,6 @@ Compaction* UniversalCompactionPicker::PickCompaction(
       assert(f->smallest_seqno <= f->largest_seqno);
       if (is_first) {
         is_first = false;
-      } else {
-        assert(prev_smallest_seqno > f->largest_seqno);
```

Comment:
after this change can `is_first` be moved down under this `if`, it's not used here it seems? It's only used in subsequent `for`,which is for levels other than L0?